tests/test_other.py

```diff
@@ -7902,9 +7902,9 @@ def test(filename, expectations, size_slack):
 
       print('test on libc++: see effects of emulated function pointers')
       test(path_from_root('tests', 'hello_libcxx.cpp'), [
-        (['-O2'], 40, ['assert'], ['waka'], 348370,  27,  220, 723), # noqa
+        (['-O2'], 40, ['assert'], ['waka'], 348370,  27,  224, 727), # noqa
         (['-O2', '-s', 'EMULATED_FUNCTION_POINTERS=1'],
-                  40, ['assert'], ['waka'], 348249,  27,  220, 723), # noqa
+                  40, ['assert'], ['waka'], 348249,  27,  224, 727), # noqa
```

Comment:
Why did this change when adding new test?

Reply:
It's the binaryen update: we used to just skip invokes with illegal signatures, but now we keep them around (but legalize them). The extra 4 added here are 4 illegal signatures that end up used in `hello_libcxx.cpp` (they just weren't called before so it happened to work).